TLM/BUILDING_INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+
+
+# Project building instructions
+
+
+### Prerequisites:
+* source code of this repository
+* source code of dependent repositories:
+  * [CSUtil.CameraControl](https://github.com/krzychu124/CSUtil.CameraControl)
+  * [OptionsFramework](https://github.com/krzychu124/OptionsFramework)
+* one prefered __IDE__ _(Integrated Development Environment)_ to build project:
+  * Visual Studio 2017 Community (free)
+  * JetBrains Raider (paid)
+  * or other similar...
+* copy contents of __CSUtil.CameraControl__ and __OptionsFramework__ to folders with corresponding names in repository source code folder ```<Your cloned source code folder>\TLM\* ```
+
+## To build project follow actions:
+
+
+Open __TMPL.sln__ located at ``` <Your cloned source code folder>\TLM\``` using preferred __IDE__.
+
+##### Visual Studio:
+
+ * use dropdown from __actions bar__ _(dropdown located under Team menu)_ to select active solution configuration
+ * fix missing libraries locations - inside __Solution Explorer__ right click on every project and select __Properties__
+ then __Reference Paths__ and add __Managed__ folder from game directory located under ```<game_dir>\Cities_Data\Managed```
+ * to build project with selected configuration choose one of actions:
+   * right click on __TLM__ project from __Solution Explorer__
+   * use _Build_ menu -> _Build Solution_ __F6__
```

Comment:
I get an error here:

![error](https://user-images.githubusercontent.com/1386719/52920106-f2eac700-3300-11e9-8547-9c747d09842a.gif)

Reply:
Approve #95 and try again 😉

Reply:
![folder](https://user-images.githubusercontent.com/1386719/52920254-71943400-3302-11e9-8582-395bc58b98b9.gif)

Reply:
LGTM

Reply:
I still get this error when building:

```File not found - CSUtil.Redirection.dll```

Reply:
Thanks, I will delete this line.
 This file would be used if we merge @VictorPhilipp 1.11.0 branch (Harmony integration)